Makefile

```diff
@@ -169,7 +169,7 @@ ifeq (,$(shell which newman))
 endif
 
 ## newman: Run Postman Newman test, the host parameter is required, and you can specify the name of the test do you wanna run (transaction, token, staking, collection, domain, healthcheck, observer). e.g $ make newman test=staking host=http//localhost
-newman: install-newman start-platform-api
```

Comment:
Why remove starting platform?

Reply:
Because we do not need to run anything here. Host -  is any endpoint, not specially the localhost

Reply:
+1